lib/features/overlays/Overlays.js

```diff
@@ -294,6 +294,9 @@ Overlays.prototype.remove = function(filter) {
 
 };
 
+Overlays.prototype.isShown = function() {
```

Comment:
Please add a test case that verifies the existence of this API.

Reply:
What I was asking for is covering the `Overlays#isShown` :see_no_evil: (`OverlaysSpec` or something). Will add the test case now. :+1:

Reply:
Added via https://github.com/bpmn-io/diagram-js/pull/719/commits/07e28f0501d6d131a2fa885ccea8b3776c57e059.

Reply:
Added another test to verify the newly added `ContextPad` API, which I renamed to `ContextPad#triggerEntry` (for clarity).

Reply:
We did not return a value anymore from `trigger`. Fixed + aadded test case to it.